app/processing/index.js

```diff
@@ -1,26 +1,17 @@
 const { processingConfig } = require('../config')
 const waitForIdleMessaging = require('../messaging/wait-for-idle-messaging')
-const schedulePendingSettlements = require('./schedule')
-const { getStatement, sendStatement, validateStatement } = require('./statement')
-const updateScheduleByScheduleId = require('./statement/update-schedule-by-schedule-id')
+const processPaymentSchedules = require('./process-payment-schedules')
+const processStatements = require('./process-statements')
 
 const start = async () => {
   try {
     if (processingConfig.constructionActive) {
       await waitForIdleMessaging()
```

Comment:
I think it's safer to do it before each processing cycle.  Just in case any new requests start coming during statement construction that would impact schedule construction.